app/js/actions/actionTypes.js

```diff
@@ -34,7 +34,7 @@ export const SET_PATIENT_DATA = 'SET_PATIENT_DATA';
 export const FETCH_CONCEPT = 'FETCH_CONCEPT';
 export const FETCH_CONCEPT_SUCCEEDED = 'FETCH_CONCEPT_SUCCEEDED';
 export const FETCH_CONCEPT_FAILED = 'FETCH_CONCEPT_FAILED';
-export const SET_ORDER_LAB_ENCOUNTER = 'SET_ORDER_LAB_ENCOUNTER';
+export const SET_LAB_RESULTS_ENCOUNTER = 'SET_LAB_RESULTS_ENCOUNTER';
```

Comment:
The Lab Results encounter and the Lab Order encounter are actually two different encounters, so I renamed this and the related actions to make it clearer.